pkg/storaged/vdo-details.jsx

```diff
@@ -105,7 +105,7 @@ export class VDODetails extends React.Component {
                         _("The creation of this VDO device did not finish and the device can't be used.")
                     </>} />
             );
-            return <StdDetailsLayout client={this.props.client} alert={broken} />;
+            return <StdDetailsLayout client={this.props.client} alerts={[broken]} />;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18979-20230706-133404-494c797e-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)